pkg/config/config.go

```diff
@@ -56,10 +56,31 @@ const (
 	DefaultVerboseLevel           = "debug"
 )
 
+type BuildInfo struct {
+	BuildTime    string `json:"os,omitempty"`
+	Os           string `json:"os,omitempty"`
+	Architecture string `json:"architecture,omitempty"`
+	Version      string `json:"version,omitempty"`
+	Revision     string `json:"revision,omitempty"`
+	Branch       string `json:"branch,omitempty"`
+}
+
 var (
+	// Note, following variables set by make
+	osys, architecture, version, revision, branch, buildTime string
+
 	instance *V3ioConfig
 	once     sync.Once
 	failure  error
+
+	BuildMetadta = &BuildInfo{
+		BuildTime:    buildTime,
+		Os:           osys,
+		Architecture: architecture,
+		Version:      version,
+		Revision:     revision,
+		Branch:       branch,
```

Comment:
I know it's a minor thing, but can we have these in the same order everywhere? 😅